bookr/src/Components/Book.js

```diff
@@ -30,41 +31,43 @@ class Book extends Component {
       return <p>Loading...</p>;
     }
 
-    const {
-      id,
-      author,
-      book_img,
-      title,
-    } = this.state.book;
+    const { id, author, book_img, title } = this.state.book;
     const { reviews } = this.state;
 
     return (
       <div className="page-container">
         <div className="book-header">
           <img src={book_img} alt={title} />
-  
+
           <div className="book-header__details">
             <h1>{title}</h1>
             <h2>By: {author}</h2>
-            <Button color="primary">Purchase Now</Button>
-            {this.props.isAuthenticated &&
-              <Link
-                to={{
-                  pathname: `/book/${id}/review`,
-                  state: this.state.book
-                }}
-              >
-                <Button
-                  outline
-                  color="primary"
+            <div className="button-wrapper">
+              <Button color="primary">Purchase Now</Button>
+              {this.props.isAuthenticated ? (
+                <Link
+                  to={{
+                    pathname: `/book/${id}/review`,
+                    state: this.state.book
+                  }}
                 >
-                  Add a Review
-                </Button>
-              </Link>
-            }
+                  <Button outline color="primary">
+                    Add a Review
+                  </Button>
+                </Link>
+              ) : (
+                <Login
+                  buttonLabel={
+                    <Button className="sign-in-button" outline color="primary">
+                      Sign in
```

Comment:
To make it more clear as to why a user should sign in when looking at a specific book, I'd do:

```suggestion
                      Sign In to Add Review
```